ui/components/component-library/picker-network/picker-network.test.js

```diff
@@ -57,7 +57,11 @@ describe('PickerNetwork', () => {
   // className
   it('should render with custom className', () => {
     const { getByTestId } = render(
-      <PickerNetwork data-testid="picker-network" className="test-class" />,
+      <PickerNetwork
+        data-testid="picker-network"
+        className="test-class"
+        label="custom"
```

Comment:
Can you revert this change on this PR and make it separately?

Reply:
Will do !